jolpica_api/ergastapi/tests/test_views.py

```diff
@@ -29,7 +29,14 @@ def test_viewsets(client: APIClient, endpoint: str, path: Path, django_assert_ma
         "http://testserver/ergast/f1/",
     )
 
-    with django_assert_max_num_queries(10):
+    if re.match(r"\d{4}.*$", endpoint):
+        max_queries = 30
```

Comment:
I'll be able to bring this down within the 10 query limit with some django magic, so don't worry about this too much.

Reply:
I'm interested to see how that works. For me, being new to django, I have no idea how that would work. Feel free to push to this branch then, if that is convenient.